docs/private-networks/tutorials/sirato-explorer.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Privacy Enabled Besu - Sirato (Epirus) Explorer
+description: Using the Sirato Explorer on a privacy enabled Besu
+---
+
+# Sirato Explorer
+
+Sirato is an EVM explorer that provides an overview over the whole network, such as block information, view contracts metadata as well as search for transactions. Sirato supports public and private networks. This tutorial will walk you through how to setup Sirato Explorer with privacy-enabled Besu network.
+
+The instructions recommends you have setup a privacy enabled-network in your local development environment. But the configuration should apply to any compatible EVM network.
+
+## Prerequisites
```

Comment:
Given the context here is using Sirato with the Quickstart, I'd put this directly in there directly under the block explorer? https://hyperledger-besu--1252.org.readthedocs.build/en/1252/private-networks/tutorials/quickstart/#block-explorer So the context is preserved. 

Add a note to say 'you must have privacy enabled' and then go to line 70 here, 'Start Sirato'

Reply:
Suggest avoiding the repetition and delete lines 12 to 69. Keep focus on Sirato from line 69 onwards?